package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "scripts": {
+    "clean": "del !.gitkeep dist/*",
```

Comment:
Done. Btw why did you list `!dist` here? Pattern `dist/*` matches children only... :thinking: